stl/inc/yvals_core.h

```diff
@@ -1148,9 +1148,9 @@
 #define __cpp_lib_char8_t 201907L
 #endif // __cpp_char8_t
 
-#if defined(__cpp_concepts) && __cpp_concepts > 201507L
+#if defined(__cpp_concepts) && __cpp_concepts > 201507L && !defined(__EDG__) // TRANSITION, EDG concepts support

```

Comment:
When we port this into MSVC we'll want to sync up with Edge folks and discuss that (1) this should unblock insertion of the updated `cpfe` nuget into the internal MSVC repo, and (2) they'll need to manually define `__cpp_lib_concepts` to override this and light up the library for their EDG-with-concepts work.

Reply:
This wasn't merged, but a variation of it was as part of #1153. I've informed the Edge team of the change and its consequences for their ongoing bringup of concepts support in Edge.